OSDOCS-13462: 4.16.37 RN (#89564)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
====

//4.16.37
[id="ocp-4-16-37_{context}"]
=== RHSA-2025:1907 - {product-title} {product-version}.37 bug fix and security update

Issued: 5 March 2025

{product-title} release {product-version}.37 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:1907[RHSA-2025:1907] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:1910[RHSA-2025:1910] advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.16.37 --pullspecs
----

[id="ocp-4-16-37-bug-fixes_{context}"]
==== Bug fixes

> * Previously, you could not create a cluster with an enabled secure proxy and certificate set in the `configuration.proxy.trustCA` field. Another issue prevented you from reaching cloud APIs through the management cluster proxy. With this release, these issues are resolved. (link:https://issues.redhat.com/browse/OCPBUGS-51296[*OCPBUGS-51296*])

Author: Not sure - due to the time crunch I'll keep it in. :) TY!

* Previously, when you deleted Dynamic Host Configuration Protocol (DHCP) network on an {ibm-power-server-title} cluster, subresources could still exist. With this release, when you delete a DHCP network, the subresources deletion now occur before continuing the destroy operation. (link:https://issues.redhat.com/browse/OCPBUGS-51111[*OCPBUGS-51111*])

* Previously, incorrect addresses were being passed to the Kubernetes EndpointSlice on a cluster, and this issue prevented the installation of the MetalLB Operator on an Agent-based cluster in an IPv6 disconnected environment. With this release, a fix modifies the address evaluation method. Red{nbsp}Hat Marketplace pods can now successfully connect to the cluster API server so that the installation of MetalLB Operator and handling of ingress traffic in IPv6 disconnected environments can occur. (link:https://issues.redhat.com/browse/OCPBUGS-50694[*OCPBUGS-50694*])

* Previously, the `cnf-tests image` used an outdated image version for running tests. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-50611[*OCPBUGS-50611*])

> * Previously, an extra name prop was being passed into resource list page extensions used to list related operands on the *CSV details* page. This caused the operand list to be filtered by the CSV name, which will almost always cause it to be an empty list. With this update, operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-46441[*OCPBUGS-46441*])

Reviewer: ```suggestion
* Previously, an extra name prop was being passed into resource list page extensions used to list related operands on the *CSV details* page. This caused the operand list to be filtered by the CSV name, which would typically cause it to be an empty list. With this update, operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-46441[*OCPBUGS-46441*])
```
